packages/agoric-cli/integration-tests/test-workflow.js

```diff
@@ -56,6 +56,7 @@ test('workflow', async t => {
     // console.error('running agoric-cli', ...extraArgs, ...args);
     return pspawnStdout(`agoric`, [...extraArgs, ...args], {
       stdio: ['ignore', 'pipe', 'inherit'],
+      // FIXME: We're running out of time under Github Actions.
```

Comment:
Now that you are no longer modifying the code to add environment variables (OMG!), is this comment now stale?